src/main/java/frc/trigon/robot/Subsystems/Roller/Roller.java

```diff
@@ -0,0 +1,91 @@
+package frc.trigon.robot.Subsystems.Roller;
+
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.*;
+
+public class Roller extends SubsystemBase {
+    private final static Roller INSTANCE = new Roller();
+    private final TalonSRX angleMotor = RollerConstants.ANGLE_MOTOR;
+    private final CANSparkMax collectionMotor = RollerConstants.COLLECTION_MOTOR;
+
+    public static Roller getInstance() {
+        return INSTANCE;
+    }
+
+    private Roller() {
+    }
+
+    public CommandBase getOpenRollerCommand()   {
```

Comment:
Add javadoc. https://www.baeldung.com/javadoc

Reply:
I will need a more depth explanation of javadoc and how you want me to use it.

Reply:
Nvm. I think I figured it out.